uwmac0073.xml

```diff
@@ -41,6 +41,12 @@
           copies of The Milwaukee Observer to the collection. Folders in box 2 were
           renumbered.</item>
       </change>
+      <change encodinganalog="583">
+        <date normal="2017-03-21">March 21, 2017</date>
+        <item>Phyllis Reske updated the finding aid after adding research 
```

Comment:
This should be at the top-- these are always done in reverse Chronological order